test-support/helpers/percy/register-helpers.js

```diff
@@ -1,7 +1,7 @@
-import Ember from 'ember';
+import { registerAsyncHelper } from '@ember/test';
 import { percySnapshot } from 'ember-percy';
 
-Ember.Test.registerAsyncHelper('percySnapshot', function(app, name, options) {
+registerAsyncHelper('percySnapshot', function(app, name, options) {
```

Comment:
@topaxi are you sure that this will work? I think `test-support` is transpiled by the host app Babel, not the one from the addon 🤔

Reply:
Oh, I did not know this, `test-support` is indeed just merged with the application tree. Thanks! 👍